utils/s3/retry_strategy.hh

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+#include <cstdint>
+#include <seastar/util/bool_class.hh>
+
+namespace aws {
```

Comment:
nit, i'd suggest add an empty line after `namespace aws` for better readability.